src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs

```diff
@@ -1,26 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.Concurrent;
```

Comment:
In this project we follow the convention of putting the using clauses inside the namespace, as it was the case before. This is obviously not really important, but I prefer it to be consistent across all files in the same project.

Reply:
Makes sense. Corrected the code here.